.github/workflows/build.yaml

```diff
@@ -24,11 +24,12 @@ jobs:
             libzstd-dev \
             meson \
             ninja-build \
+            openssl-dev \
```

Comment:
The package isn't `openssl-dev`

Looks like the workflow that got run isn't the one in the PR, so it's now failing.